### PR TITLE
Proposed code standard (Standard)

### PR DESCRIPTION
**Pull request by samtarling**
Set up package.json with sane values, do a `npm install` and save -lock
Configure a `.eslintrc.json` using the Standard standard
Ran `eslint spihelper.js --fix`
~~Still failing~~

Related #57

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+    "env": {
+        "browser": true,
+        "es2021": true
+    },
+    "extends": [
+        "standard"
+    ],
+    "parserOptions": {
+        "ecmaVersion": 12
```
Comment (GeneralNotability) [inline .eslintrc.json:10]: Unless you have a specific need for 12 (other than "it's the latest"), I'd prefer 8/2017 - that's the standard I've been targeting. If you do want something from >= 8, we can do that, I'll just need to make a deprecation plan.

**Review by GeneralNotability — COMMENTED** [review]
Explain to me again why we're so opposed to semicolons? I don't absolutely need them, but this is a lot of churn.

**Comment by samtarling** [comment]
> Explain to me again why we're so opposed to semicolons? I don't absolutely need them, but this is a lot of churn.

I _personally_ prefer semicolons, coming from a PHP background - but as Standard put it:

"At the end of the day you have to 'just pick something', and that's the whole philosophy of standard -- its a bunch of sensible 'just pick something' opinions."

For a more in-depth answer, they link to a blog post which is fairly good at https://standardjs.com/rules.html#semicolons

File: .eslintrc.json (hunk 1 above)
Comment (samtarling) [reply .eslintrc.json:10]: No specific need - most of this was auto-generated.

Will change, re-run and upload

File: package.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "spihelper",
+  "version": "2.5.4",
```
Comment (samtarling) [inline package.json:3]: Could you confirm the version number we're at?

Comment (GeneralNotability) [reply package.json:3]: 2.5.4 is correct - wrong in the spihelper header, but 2.5.4 is the latest tag.

**Pull request merged**